src/cljs/zetawar/data.cljs

```diff
@@ -133,6 +136,7 @@
                 :cost 75
                 :movement 9
                 :can-capture true
+                :can-repair false
```

Comment:
It works, but the work I did in subs.cljs might seem a little janky. There's probably a better way to handle the selected and targeted values, but I'm not sure what that is.